apm-agent-plugins/apm-spring-webflux/apm-spring-webflux-testapp/pom.xml

```diff
@@ -37,13 +37,6 @@
     </dependencyManagement>
 
     <dependencies>
-
-        <dependency>
-            <groupId>co.elastic.apm</groupId>
-            <artifactId>apm-agent-api</artifactId>
-            <version>${project.version}</version>
-        </dependency>
-
```

Comment:
The Webflux webapp is stand-alone, thus should not have dependencies on the agent, thus using the API was required for that.

Reply:
We decided that for now we will keep it as is to make sure we continuously test recycling at all cases.
We can get back to this once there is a need to run as a standalone app.